prez/services/listings.py

```diff
@@ -214,7 +205,7 @@ async def ogc_features_listing_function(
                 TriplesSameSubjectPath.from_spo(*innser_select_triple)
             )
             subselect_kwargs["inner_select_vars"] = [queryable_var]
-            subselect_kwargs["limit"] = 100
+            subselect_kwargs["limit"] = settings.listing_count_limit
```

Comment:
Where does this setting get set? Why was it hardcoded to 100 before? What happens if this setting is not set, is there a fallback?

Reply:
In `prez/config.py`, it's the same as other settings using pydantic base settings so can be set with the capitalised env var of the same name. https://github.com/RDFLib/prez/blob/main/README.md#listing-and-search-configuration

It was hardcoded while being developed and left in accidentally